packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-component-factory-creator.ts

```diff
@@ -59,93 +54,20 @@ function watchComponentFactoryCreator() {
     return project.componentsPath;
   });
 
-  watchItems([componentRootPath, ...projectComponentsPaths], writeComponentFactoryCreator);
+  const componentFactoryCreatorWrapper = () => 
+  {
+    writeComponentFactoryCreator(config.componentRootPath, config.projectRootPath);
+  }
+ 
+  watchItems([config.componentRootPath, ...projectComponentsPaths], componentFactoryCreatorWrapper);
 }
 
 /**
- * Generates the component factory creator file and saves it to the filesystem.
- * By convention, we expect to find: 
- *   * React components under src/components/** 
- *   * Project components under src/projects/<project>/components/**
- * (subfolders are searched recursively). 
- * The filename, with the extension stripped, is used for the component's
- * string name (for mapping to Sitecore). The filename, with extension and non-word characters
- * stripped, is used to identify the component's JavaScript module definition (for initializing
- * new component instances in code).
- * Modify this function to use a different convention.
+ * Run watch mode, watching on @var paths
  */
-export function writeComponentFactoryCreator() {
-  /**
-   * You can specify components which you want to import from external/internal packages
-   * in format:
-   *  {
-   *    name: 'package name',
-   *    components: [
-   *      {
-   *        componentName: 'component name', // component rendering name,
-   *        moduleName: 'module name' // component name to import from the package
-   *      }
-   *    ]
-   *  }
-   */
```

Comment:
Looks like we lost the ability to specify your own packages with this move? Should consider adding another option to the `writeComponentFactoryCreator` which accepts these.